### PR TITLE
[General] Improve navigation on touchscreen and gamepad and revert some colours

### PR DESCRIPTION
**Pull request by wiryfuture**
Makes clickable area for checkboxes far bigger so it's easier to toggle on touchscreens like the steam deck (and just in general)

before: hover and click does nothing, have to go on checkbox
Screenshot_20220206_235715
Screenshot_20220206_235811

after: click text and it works (well, technically, the entire 'row' beside the checkbox) 
Screenshot_20220207_000024
Screenshot_20220207_000049

### CODE REVIEW

**Comment by flavioislima** [comment]
Tested here, looks good. The only missing color now is the Download icon on both the Game Card and the Wine Manager.
they need to be `--seconday` as well.

**Comment by wiryfuture** [comment]
> Tested here, looks good. The only missing color now is the Download icon on both the Game Card and the Wine Manager. they need to be `--seconday` as well.

done!

File: src/screens/Settings/components/OtherSettings/index.tsx [hunk 1]
```diff
@@ -306,23 +321,23 @@ export default function OtherSettings({
               onChange={handleLauncherArgs}
             />
           </span>
+          <InfoBox text="infobox.help">
```
Comment (arielj) [inline src/screens/Settings/components/OtherSettings/index.tsx:324]: this InfoBox code is duplicated, maybe you can store the element in a variable before the conditionals and then do `{infoBox}`

File: src/screens/Settings/index.tsx [hunk 2]
```diff
@@ -343,39 +343,53 @@ function Settings() {
   return (
     <>
       <div className="Settings">
-        <div className="settingsNavbar">
+        <nav role={'list'} className="settingsNavbar">
```
Comment (arielj) [inline src/screens/Settings/index.tsx:346]: you can do `role='list'` without the curly braces

File: src/components/UI/LanguageSelector/index.tsx [hunk 3]
```diff
@@ -99,7 +99,7 @@ export default function LanguageSelector({
     <select
       data-testid="languageSelector"
       onChange={(event) => handleLanguageChange(event.target.value)}
-      className={className}
+      className={className + ' is-drop-down'}
```
Comment (arielj) [inline src/components/UI/LanguageSelector/index.tsx:102]: are these `is-drop-down` classes needed? I see they are added to select tags which are, by default, drop downs, isn't the `.settingSelect` selector enough?

**Review by arielj — COMMENTED** [review]
I haven't tested it locally, but I added some code style comments

File: src/components/UI/LanguageSelector/index.tsx (hunk 3 above)
Comment (wiryfuture) [reply src/components/UI/LanguageSelector/index.tsx:102]: Needed for the cursor to change appropriately

Comment (flavioislima) [inline src/components/UI/LanguageSelector/index.tsx:102]:
```suggestion
      className={`${className} is-drop-down`}
```

**Review by flavioislima — DISMISSED** [review]
Looks good overall. Left a small comment only but I believe is good to be merged.

File: src/components/UI/LanguageSelector/index.tsx (hunk 3 above)
Comment (wiryfuture) [reply src/components/UI/LanguageSelector/index.tsx]: bruh package to join classes is that like isOdd and isEven

Comment (wiryfuture) [reply src/components/UI/LanguageSelector/index.tsx]: > 

done

**Review by flavioislima — APPROVED** [review]
Great job!

**Pull request merged**